pkg/template/template_test.go

```diff
@@ -54,7 +59,8 @@ func TestWriteChangelogDestinationOpenError(t *testing.T) {
 	outputFile := "doesnotexist/foo"
 	testObj := struct{}{}
 
-	err := WriteChangelog("testdata/test.tmpl", testObj, outputFile)
+	tmpl := New("testdata")
+	err := tmpl.WriteChangelog("test.tmpl", testObj, outputFile)
 	if !assert.Error(t, err) {
 		return
 	}
```

Comment:
Nit: We could probably cut down on code reuse with an array of test cases that we iterate over

Reply:
Good point - this might be something too look into in the next iteration.